understanding/label-in-name.html

```diff
@@ -273,82 +190,61 @@ <h4>Mathematical expressions and formulae</h4>
                <section id="accessible-name-and-description-computation-specification">
                   
                   
-                  <h3>Accessible Name and Description Computation specification</h3>
+                  <h3>Accessible Name and Description Computation 仕様</h3>
                   
-                  <p>It is important to understand how the accessible name is derived. The <a href="https://www.w3.org/TR/accname-1.1/">Accessible Name and Description Computation 1.1</a> and the <a href="https://w3c.github.io/html-aam/#accessible-name-and-description-computation">HTML Accessibility API Mappings 1.0</a> describe how the accessible name is computed, including which attributes are considered
-                     in its calculation, and in what order of preference. If a component has multiple possible
-                     attribute values that could be used for its accessible name, only the most preferred
-                     of those values will be computed. None of the other, less preferred values will be
-                     part of the name. For the most part, existing established programmatic relationships
-                     between labels and controls are reinforced by the specification.
+                  <p>アクセシブルな名前 (accessible name) がどのように派生するかを理解することが重要である。<a href="https://www.w3.org/TR/accname-1.1/">Accessible Name and Description Computation 1.1</a> 及び <a href="https://w3c.github.io/html-aam/#accessible-name-and-description-computation">HTML Accessibility API Mappings 1.0</a> は、アクセシブルな名前 (accessible name) の計算方法を説明する。これには、計算で考慮される属性や優先順位などが含まれる。コンポーネントに、アクセシブルな名前 (accessible name) に使用できる複数の属性値がある場合、それらの値の中で最も優先されるものだけが計算される。他のあまり好ましくない値は名前 (name) の一部にはならない。ほとんどの場合、ラベルとコントロールの間に確立された既存のプログラム上の関係が仕様によって強化されている。
                   </p>
                   
-                  <p>Other text displayed on the screen that is correctly coded to meet 1.3.1: Info and
-                     Relationships is <strong>not</strong> normally factored into the calculation for the accessible name of a UI component
-                     without author intervention (via ARIA labeling techniques). The most common of these
-                     are:
+                  <p>1.3.1: 情報及び関係性を満たすように正しくコーディングされた画面に表示されるその他のテキストは、通常、作成者の介入なしに（ARIAラベル付け手法を通して）UI コンポーネントのアクセシブルな名前 (accessible name) の計算に考慮<strong>されない</strong>。これらの中で最も一般的なものは次のとおり:
```

Comment:
「正しくコーディングされた画面」などと読めてしまい、ちょっとわかりにくいような。こんな感じにしてはどうでしょうか。

```suggestion
                  <p>画面に表示されるその他のテキストが、1.3.1: 情報及び関係性を満たすように正しくコーディングされていても、通常、（ARIA のラベル付け手法によって）作成者が介入しない限り、UI コンポーネントのアクセシブルな名前 (accessible name) の計算には影響<strong>しない</strong>。これらの中で最も一般的なものは次のとおり:
```